eng/scripts/Language-Settings.ps1

```diff
@@ -4,15 +4,15 @@ $PackageRepository = "Nuget"
 $packagePattern = "*.nupkg"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/dotnet-packages.csv"
 
-function Extract-dotnet-PkgProperties ($pkgPath, $serviceName, $pkgName)
+function Get-dotnet-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
```

Comment:
I assume these changes are needed in order to switch the verify-changelog logic you have at https://github.com/Azure/azure-sdk-tools/pull/823/files#diff-9bb641df46628fac1b924b416354af63R10. In which case I suggest splitting this PR (and the other ones in the other languages) to have only the language-settings first then get the tools changes in that are additive and then have another PR that renames ServiceName to ServiceDirectory in the template usages.